bundles/org.openhab.binding.openwebnet/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -343,6 +343,22 @@
 		<state readOnly="true" pattern="%.0f %unit%"></state>
 	</channel-type>
 
+	<channel-type id="energyThisMonth">
+		<item-type>Number:Energy</item-type>
+		<label>Energy this month</label>
```

Comment:
Energy This Month
Please also adjust the i18n file.

Reply:
ok done. Energy today also updated

Reply:
No, undone. I was requesting to use camel came for the label:
`<label>Energy This Month</label>`

Reply:
ok